code/game/objects/items/weapons/clown_items.dm

```diff
@@ -134,4 +134,28 @@
 	desc = "Damn son, where'd you find this?"
 	icon_state = "air_horn"
 	honksound = 'sound/items/AirHorn2.ogg'
-	cooldowntime = 50
\ No newline at end of file
+	cooldowntime = 50
+
+/obj/item/weapon/bikehorn/golden
+	name = "golden bike horn"
+	desc = "Golden? Clearly, its made with bananium! Honk!"
+	icon_state = "gold_horn"
+	item_state = "gold_horn"
+
+/obj/item/weapon/bikehorn/golden/attack()
+	flip_mobs()
+	return ..()
+
+/obj/item/weapon/bikehorn/golden/attack_self()

```

Comment:
Still, people could be confused by this and think the proc doesn't take any argument.